.github/pull_request_template.md

```diff
@@ -1,24 +1,28 @@
-### Name of the story
+# A nice friendly title for what this PR does
 
-[Airtable ticket](https://airtable.com/####)
+Relates to issue #(an issue)/bug found due to #(previous pull request).
```

Comment:
Might be nice to start with a label for what this is, e.g. `Short description of what the PR covers:...`

Reply:
How is it now?

Reply:
Great.